### ISSUE TITLE
Details Button not keyboard accessible

### ISSUE THREAD

**Issue by bmuenzenmeyer** [nodejs/nodejs.org#8048]
### URL:

https://nodejs.org/en/about/previous-releases

### Browser Name:

Chrome

### Browser Version:

138.0.7204.169 (Official Build) (64-bit)

### Operating System:

Windows 10

### How to reproduce the issue:

1. Navigate to https://nodejs.org/en/about/previous-releases
2. tab to table within https://nodejs.org/en/about/previous-releases#looking-for-the-latest-release-of-a-version-branch
3. cannot tab to any of the entries `Details`

<img width="766" height="921" alt="Image" src="https://github.com/user-attachments/assets/e4a879d9-20e1-4bda-bf88-b5a56f46bc97" />

> discovered during #7990 testing, but happening in production now too
